presto-spi/src/main/java/com/facebook/presto/spi/block/DictionaryBlock.java

```diff
@@ -401,6 +401,11 @@ Slice getIds()
         return Slices.wrappedIntArray(ids, idsOffset, positionCount);
     }
 
+    int[] getRawIds()
```

Comment:
public

Reply:
@mbasmanova The original getIds() was not public:

```
Slice getIds()
    {
        return Slices.wrappedIntArray(ids, idsOffset, positionCount);
    }
```
Also it appeared before some of the public methods. I was wondering if we should move both to the end. It looks like it doesn't need to be public for now.

Reply:
@yingsu00 Got it. Thanks.